tools/tool-testing/galaxy-utils.js

```diff
@@ -0,0 +1,251 @@
+var _ = require('underscore');
+var process = require('process');
+var selftest = require('../tool-testing/selftest.js');
+var Run = selftest.Run;
+var testUtils = require('../tool-testing/test-utils.js');
+var files = require('../fs/files.js');
+var utils = require('../utils/utils.js');
+var authClient = require('../meteor-services/auth-client.js');
+var auth = require('../meteor-services/auth.js');
+
+// Run curl with the given specifications. Return an instance of Run.
+var runCurl = function (/*args*/) {
+  var curl = "/usr/bin/curl";
+  return new Run(curl, {
+    args: _.toArray(arguments)
+  });
+};
+
+// Some constants.
+var GALAXY_USERNAME = process.env.GALAXY_USERNAME;
+var GALAXY_PASSWORD = process.env.GALAXY_PASSWORD;
+var GALAXY_URL = process.env.DEPLOY_HOSTNAME;
+
+// Fail if the test is obviously not set up for using Galaxy.
+//
+// Make sure that we have at least set all of the variables that we need to run
+// against a Galaxy (GALAXY_URL, username & password). An extra safety
+// check to avoid strange errors, deploying/calling random methods on Mother,
+// etc.
+exports.sanityCheck = selftest.markStack(function () {
```

Comment:
Yeah, I was planning to put the docs on one of the Galaxy hackpads (ex: Admining Galaxy & Process: Releasing Galaxy). I am not really sure where to put them, to be honest. Many of the usual places (ex: the Meteor wiki) are clearly incorrect. Any suggestions?